AppLovinMAXDemo/Plugins/AppLovinMAX/Source/AppLovinMAX/AppLovinMAX.Build.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright AppLovin Corporation. All Rights Reserved.
+
+using UnrealBuildTool;
+using System.IO;
+
+public class AppLovinMAX : ModuleRules
+{
+	private const string PluginVersion = "1.0";
+
+	public AppLovinMAX(ReadOnlyTargetRules Target) : base(Target)
+	{
+		PCHUsage = ModuleRules.PCHUsageMode.UseExplicitOrSharedPCHs;
+
+		PublicIncludePaths.AddRange( new string[] {} );
+		PrivateIncludePaths.AddRange( new string[] { "AppLovinMAX/Private" } );
+		PrivateIncludePathModuleNames.AddRange( new string[] { "Settings" } );
+		PublicDependencyModuleNames.AddRange( new string[] { "Core" } );
+		PrivateDependencyModuleNames.AddRange(
+			new string[]
+			{
+				"CoreUObject",
+				"Engine",
+			}
+		);
+		
+		DynamicallyLoadedModuleNames.AddRange( new string[] { } );
+		
+		if ( Target.Platform == UnrealTargetPlatform.IOS )
+		{
+			string AppLovinIOSPath = Path.Combine( ModuleDirectory, "..", "ThirdParty", "IOS" );
+			string AppLovinSDKPath = Path.Combine( AppLovinIOSPath, "AppLovinSDK.embeddedframework.zip" );
+			string AppLovinPluginPath = Path.Combine( AppLovinIOSPath, "MAX_Unreal_Plugin.embeddedframework.zip" );
+			if ( File.Exists( AppLovinSDKPath ) && File.Exists( AppLovinPluginPath ) )
+			{
+				System.Console.WriteLine( "AppLovin IOS Plugin found" );
+				
+				// Add the AppLovin framework
+				PublicAdditionalFrameworks.Add(
+					new Framework(
+						"AppLovinSDK",
+						AppLovinSDKPath
+					)
+				);
+
+				// Add the AppLovin Unreal iOS plugin
+				PublicAdditionalFrameworks.Add(
+					new Framework(
+						"MAX_Unreal_Plugin",
+						AppLovinPluginPath
+					)
+				);
+
+				PublicFrameworks.AddRange(
```

Comment:
Might be worth adding whatever we have [here](https://dash.applovin.com/documentation/mediation/manual-integration-ios#add-frameworks)... Also, I've added `CFNetwork` to the docs in the next docs push.